src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Acquisition/NetCoreRuntimeVersionsRegistryReader.cs

```diff
@@ -7,8 +7,8 @@ namespace Microsoft.VisualStudio.ProjectSystem.VS.Acquisition;
 
 internal sealed class NetCoreRuntimeVersionsRegistryReader
 {
-    private static readonly string s_arm64NetCoreRegistryKeyPath = "SOFTWARE\\dotnet\\Setup\\InstalledVersions\\ARM64\\sharedfx\\";
-    private static readonly string s_x64NetCoreRegistryKeyPath = "SOFTWARE\\WOW6432Node\\dotnet\\Setup\\InstalledVersions\\x64\\sharedfx\\";
+    private static readonly string s_arm64NetCoreRegistryKeyPath = """SOFTWARE\dotnet\Setup\InstalledVersions\ARM64\sharedfx\""";
+    private static readonly string s_x64NetCoreRegistryKeyPath = """SOFTWARE\WOW6432Node\dotnet\Setup\InstalledVersions\x64\sharedfx\""";
```

Comment:
Verbatims strings would work here fine. I'm trying to use raw strings instead of verbatim strings as they can do everything verbatim strings can and more, and it's nice to use fewer of C#'s many features. The overhead of a few extra `"` characters doesn't seem too bad to me. To me it makes it clearer that something special is going on. What are your thoughts?